Psychtoolbox/PsychDemos/PsychTutorials/AlphaImageTutorial.m

```diff
@@ -18,8 +18,8 @@
 
```

Comment:
The commit message should contain the name of the function you modified ie.,
AlphaImageTutorial: Update to use more modern 0 - 1 color range.